lib/pebble.js

```diff
@@ -104,9 +105,23 @@ function addExtraData (first, req, sbx) {
     }
   }
 
+  function addPumpIOB () {
+    if (req.pumpiob) {
+      var result = pumpiob.getPumpIOB(
+        data,
+        sbx.withExtendedSettings(pumpiob).extendedSettings,
+        Date.now()
+      );
+      if (result) {
+        first.iob = result.pebbleDisplay;
+      }
+    }
+  }
+
   addDelta();
   addBattery();
   addIOB();
+  addPumpIOB();
```

Comment:
It sets the same key as the existing Pebble IOB code, and I think this is a big enough win that it's worth it. (A lot of people already using NS with Dexcom say they're getting a Connect just for this kind of data.)